qutebrowser/browser/history.py

```diff
@@ -113,13 +113,20 @@ class CompletionHistory(sql.SqlTable):
 
     """History which only has the newest entry for each URL."""
 
-    def __init__(self, parent=None):
-        super().__init__("CompletionHistory", ['url', 'title', 'last_atime'],
+    FRECENCY_BONUS = 43200  # Seconds in 12 hours
```

Comment:
It would be quite easy to do so but it looks to me like too obscure to make it a setting. Anyway, it doesn't hurt to make it a setting. If you don't get, don't touch it :)